extensions/prototypes/labgraph_monitor/README.md

```diff
@@ -0,0 +1,66 @@
+# LabGraph Monitor
+
+LabGraph Monitor helps visualize the graph built. Inorder to run the application, you will need to first set it up using React using the below steps:
+
+First, incase you do not have npm installed, follow the below steps to get started:
```

Comment:
"incase" -> "in case".

I'd also remove the "First, " part since first is used in the prior sentence.